docs/reference/index-modules/indexing-pressure.asciidoc

```diff
@@ -0,0 +1,70 @@
+[[index-modules-indexing-pressure]]
+== Indexing pressure
+
+Indexing documents into Elasticsearch introduces system load in the form of
+memory and CPU load. Each indexing operation includes coordinating, primary, and
+replica stages. These stages can be performed across multiple nodes in the
+cluster. If too much indexing work is introduced into the system, the cluster
+can become saturated. This can adversely impact other components of the system
+such as searches, cluster coordination, background processing, etc.
+
+Indexing pressure is primarily generated by external operations such as indexing
+requests or internally by mechanisms such recoveries and cross-cluster
+replication.
+
+Elasticsearch internally monitors indexing load. When the load exceeds
+certain limits, new indexing work will be rejected.
+
```

Comment:
I re-ordered this to try to improve the flow.
```suggestion
Indexing documents into {es} introduces system load in the form of memory and
CPU load. Each indexing operation includes coordinating, primary, and replica
stages. These stages can be performed across multiple nodes in a cluster.

Indexing pressure can build up through external operations, such as indexing
requests, or internal mechanisms, such as recoveries and {ccr}. If too much
indexing work is introduced into the system, the cluster can become saturated.
This can adversely impact other operations, such as search, cluster
coordination, and background processing.

To prevent these issues, {es} internally monitors indexing load. When the load
exceeds certain limits, new indexing work is rejected.

```